_pages/survey.md

```diff
@@ -59,29 +59,29 @@ onload="if(submitted) {window.location='/thank-you/';}"></iframe>
   
        <label><strong>4)Choose the answer that best represents your feelings in response to the statement ‘Economics research occurs too much in silos, with academic researchers working on their own personal project rather than collaborating on larger longer-term projects’ </strong></label><br>
       <input type="radio" id="4.1" name="entry.2013966563" value="Agree Strongly">
-      <label for = "4.1">Agree Strongly</label><br>
+      <label for = "4.1">Strongly Agree</label><br>
       <input type="radio" id="4.2" name="entry.2013966563" value="Agree">
       <label for = "4.2">Agree</label><br>
       <input type="radio" id="4.3" name="entry.2013966563" value="Neutral">
       <label for = "4.3">Neutral</label><br>
       <input type="radio" id="4.4" name="entry.2013966563" value="Disagree">
       <label for = "4.4">Disagree</label><br>
       <input type="radio" id="4.5" name="entry.2013966563" value="Disagree Strongly">
-      <label for = "4.5">Disagree Strongly</label><br><br>
+      <label for = "4.5">Strongly Disagree</label><br><br>
```

Comment:
Not sure if the "Disagree Strongly" ordering is a British thing but the original design for a Likert scale like the one you're using has it the other way around